py/desispec/workflow/desi_proc_funcs.py

```diff
@@ -331,9 +331,25 @@ def determine_resources(ncameras, jobdesc, queue, nexps=1, forced_runtime=None,
         ncores          = 20 * (10*(ncameras+1)//20) # lowest multiple of 20 exceeding 10 per camera
         ncores, runtime = ncores + 1, 45             # + 1 for worflow.schedule scheduler proc
     elif jobdesc in ('FLAT', 'TESTFLAT'):
-        ncores, runtime = 20 * nspectro, 25
+        runtime = 25
+        if system_name[0:10] == 'perlmutter':
+            ncores = config['cores_per_node']
+        else:
+            ncores = 20 * nspectro
+    elif jobdesc in ('TILENIGHT'):
```

Comment:
Minor: this was probably intended to be `('TILENIGHT',)` to be a tuple, rather than `('TILENIGHT')` where the parentheses are just interpreted as operator precedence not a tuple.  It happens that `'TILENIGHT' in 'TILENIGHT'` evaluates to `True` anyway so it works with or without that trailing comma inside the parentheses, but that is a lucky accident of how strings work with "in".

In contrast, this doesn't work with other types, e.g, `some_integer in (1)` raises a TypeError, while `some_integer in (1,)` works, and it is arguably preferable to use `if some_integer == 1` if you really just have a single element to prevent confusion about how those parentheses and trailing commas are parsed (which has bitten us in other cases which is why I noticed it here and wrote a paragraph about it...)

Reply:
@sbailey I changed all the `in (x)` to `== x` in this block.